.buildkite/pipeline.yml

```diff
@@ -14,150 +50,171 @@ env:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - name: "Prometheus compatibility (:docker:)"
-    command: make clean install-vendor-m3 docker-compatibility-test
-    parallelism: 1
-    env:
-      CGO_ENABLED: 0
-      GIMME_GO_VERSION: 1.18.x
-    plugins:
-        gopath-checkout#v1.0.1:
-          import: github.com/m3db/m3
-    <<: *common
-  - name: "Integration (dbnode Recently Read) %n"
-    parallelism: 2
-    command: make clean install-vendor-m3 test-ci-integration-dbnode cache_policy=recently_read
-    plugins:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - name: "Integration (dbnode LRU) %n"
-    parallelism: 2
-    command: make clean install-vendor-m3 test-ci-integration-dbnode cache_policy=lru
-    plugins:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - name: "Integration (aggregator TCP client) %n"
-    parallelism: 1
-    command: make clean install-vendor-m3 test-ci-integration-aggregator aggregator_client=tcp
-    plugins:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - name: "Integration (aggregator m3msg client) %n"
-    parallelism: 1
-    command: make clean install-vendor-m3 test-ci-integration-aggregator aggregator_client=m3msg
-    plugins:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - label: "Integration (m3em, cluster, msg, metrics) %n"
     parallelism: 4
-    command: make clean install-vendor-m3 test-ci-integration-m3em test-ci-integration-cluster test-ci-integration-msg test-ci-integration-metrics
     plugins:
       docker-compose#v2.5.1:
         run: app
         workdir: /go/src/github.com/m3db/m3
-    <<: *common
-  - name: "Documentation tests"
-    command: make clean install-vendor-m3 docs-test
-    env:
-      CGO_ENABLED: 0
-      GIMME_GO_VERSION: 1.18.x
-    plugins:
-        gopath-checkout#v1.0.1:
-          import: github.com/m3db/m3
-    <<: *common
-  - label: "FOSSA license scan"
-    command: make clean install-vendor-m3 fossa
-    plugins:
-      docker-compose#v2.5.1:
-        run: app
-        workdir: /go/src/github.com/m3db/m3
-        env:
-          - FOSSA_API_KEY
-    <<: *common
-  - name: "Check for docker and docs builds :docker: :books:"
-    commands:
-    - ".ci/docker/check_do_docker.sh"
-    - ".buildkite/scripts/check_do_docs.sh"
-    agents:
-      queue: init
-    timeout_in_minutes: 10
-    retry:
-      automatic:
-        limit: 1
-      manual: true
+      kubernetes:
+        <<: *kubernetes
+        podSpec:
+          <<: *podSpec
+          containers:
+            - <<: *commandContainer
+              command:
+                - |-
+                  make clean install-vendor-m3 test-base
+    <<: *common
+#  - name: "Big Unit %n"
```

Comment:
@kentzeng12 can you add a big comment block over this saying something to the effect of "Build is currently broken; we're disabling these until it's fixed".

Link in a github issue for it as well (create one and assign to yourself)